static/css/pages/token.css

```diff
@@ -319,7 +319,7 @@
 
   .tokens-page .app-showcase .showcase-content {
     max-width: 430px;
-    margin-left: 40px;
+    margin-left: 40px !important;
```

Comment:
Is this `!important` necessary? Can we use a more specific selector to avoid this?

Reply:
Fixed